R/as_factor.R

```diff
@@ -34,7 +34,12 @@ as_factor <- function(x, ...) {
 #' @rdname as_factor
 #' @export
 as_factor.factor <- function(x, ...) {
-  x
+  structure(
+    x,
+    label = attr(x, "label", exact = TRUE),
```

Comment:
Can you please order these alphabetically?

Reply:
done